ranger-zookeeper/src/main/java/com/flipkart/ranger/zookeeper/ServiceProviderBuilders.java

```diff
@@ -16,8 +16,7 @@
 
 package com.flipkart.ranger.zookeeper;
 
-import com.flipkart.ranger.zookeeper.zk.ZkServiceProviderBuilder;
-import com.flipkart.ranger.core.finder.unsharded.UnshardedClusterInfo;
+import com.flipkart.ranger.zookeeper.serviceprovider.ZkServiceProviderBuilder;
```

Comment:
Remove constructor below and put `@UtilityClass`

Reply:
Part of the lombok cleanup